Jenkinsfile_CNP

```diff
@@ -35,14 +35,15 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
   ]
 }
 
+def yarnBuilder = new uk.gov.hmcts.contino.YarnBuilder(this)
+
 withPipeline("nodejs", product, component) {
     loadVaultSecrets(secrets)
```

Comment:
are there any secrets that are no longer needed anymore? I wouldn't be surprised if none are needed

Reply:
Yes, they are needed. Deleted them and added one by one functional tests failed